src/script/components/calling/CallParticipantsListItem/CallParticipantItemContent/CallParticipantItemContent.styles.ts

```diff
@@ -75,15 +75,18 @@ export const contentText: CSSObject = {
   fontWeight: 'var(--font-weight-medium)',
 };
 
-export const nameWrapper: CSSObject = {
-  color: 'var(--main-color)',
+export const nameWrapper = (isAudioEstablished: boolean): CSSObject => ({
+  color: isAudioEstablished ? 'var(--main-color)' : 'var(--gray-70)',
```

Comment:
```suggestion
  color: isAudioEstablished ? 'var(--main-color)' : 'var(--text-input-placeholder)',
```
light/dark mode css variable ;)